lerobot-web-fe/src/components/ChatInput.module.css

```
@@ -0,0 +1,98 @@
+.chatForm {
+  padding: 9px 16px;
+  box-shadow: 0 0 9px 0 #00000088;
+  background-color: #00000030;
+  pointer-events: all;
+  color: #333;
+  background-color: #f0f0f091;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background-color is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)